website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -95,23 +96,31 @@ When using a CLI workflow, local environment variables prefixed with `TF_VAR_` (
 
 Workspace-specific variables always overwrite variables from variable sets that have the same key. Refer to [overwrite variables from variable sets](/terraform/cloud-docs/workspaces/variables/managing-variables#overwrite-variable-sets) for details.
 
-### 4. Non-Global Variable Sets
+### 4. Workspace-Scoped Variable Sets
 
-Non-global variables are only applied to a subset of workspaces in an organization.
+Variables in workspace-scoped variable sets are only applied to a subset of workspaces in an organization.
 
-When non-global variable sets have conflicting variables, Terraform Cloud compares the variable set names and uses values from the variable set with lexical precedence. Terraform and Terraform Cloud operate on UTF-8 strings, and Terraform Cloud sorts variable set names based the on lexical order of Unicode code points.
+When workspace-scoped variable sets have conflicting variables, Terraform Cloud compares the variable set names and uses values from the variable set with lexical precedence. Terraform and Terraform Cloud operate on UTF-8 strings, and Terraform Cloud sorts variable set names based the on lexical order of Unicode code points.
 
 For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, Terraform Cloud will use any conflicting variables from `A_Variable_Set`. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the lexical ordering of variable set names when determining precedence.
 
-### 5. Global Variable Sets
+### 5. Project-Scoped Variable Sets
 
-Non-global variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. Terraform does not allow global variable sets to contain variables with the same key, so they cannot conflict.
+Variables in project-scoped variable sets are only applied to a subset of workspaces in a project.
 
-### 6. `*.auto.tfvars` Variable Files
+When project-scoped variable sets have conflicting variables, Terraform Cloud compares the variable set names and uses values from the variable set with lexical precedence. Terraform and Terraform Cloud operate on UTF-8 strings, and Terraform Cloud sorts variable set names based the on lexical order of Unicode code points.
+
+For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, Terraform Cloud will use any conflicting variables from `A_Variable_Set`. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the lexical ordering of variable set names when determining precedence.
```

Comment:
I decided to duplicate parts of `Workspace-Scoped Variable Sets` because a user may directly go to `Project-Scoped Variable Sets` without having read `Workspace-Scoped Variable Sets`. That being said, happy to change if you have suggestions!

Reply:
This was a great idea!